Velaptor/Input/Exceptions/NoKeyboardException.cs

```diff
@@ -2,10 +2,10 @@
 // Copyright (c) KinsonDigital. All rights reserved.
 // </copyright>
 
-using System;
-
 namespace Velaptor.Input.Exceptions;
 
+using System;
+
 /// <summary>
 /// Occurs when no keyboard has been detected in the system.
```

Comment:
Sentence structure.  Both are correct, but my suggestion flows better. 
```suggestion
/// Occurs when a keyboard has not been detected in the system.
```